contract/.cargo/config.toml

```diff
@@ -1,2 +1,5 @@
 [build]
 target = "wasm32-unknown-unknown"
+
+[target.wasm32-unknown-unknown]
+rustflags = ["-C", "link-arg=-s"]
```

Comment:
What's the intent with this?

Reply:
This just minimizes the amount of bloat within the built wasm file